src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/NewAuthModal.java

```diff
@@ -0,0 +1,54 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+/**
+ * @ownership Social
+ */
+public class NewAuthModal {
```

Comment:
Change the class name

The auth modal may be new now, but it will not be new after one year
Putting relative time references into names of entities is a bad practice and confuses people as time passes